var/spack/repos/builtin/packages/gconf/package.py

```diff
@@ -7,9 +7,13 @@
 
 
 class Gconf(AutotoolsPackage):
-    """GConf is a system for storing application preferences."""
+    """GConf is a system for storing application preferences.
 
-    homepage = "https://projects.gnome.org/gconf/"
+    Note that this software is now deprecated in favor of moving to GSettings
```

Comment:
For these deprecated packages, should we add `deprecated=True` to each version?

Reply:
Sure thing, all versions?

Reply:
Yep. If the package has a maintainer, just ping them first.